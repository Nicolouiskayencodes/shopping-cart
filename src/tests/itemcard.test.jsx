import { describe, it, expect } from "vitest";
import { render, screen } from "@testing-library/react";
import ItemCard from "../components/itemcard";
describe("Cart component", () => {
  it("renders price", () => {
    render(<ItemCard props={{image: "url", title: "title", price: 10}}/>);
    expect(screen.getByText("$10")).toBeInTheDocument();
  });
  it("renders card", ()=>{
    const { container } = render(<ItemCard props={{ image: "url", title: "title", price: 10}}/>);
    expect(container).toMatchSnapshot();
  })
});