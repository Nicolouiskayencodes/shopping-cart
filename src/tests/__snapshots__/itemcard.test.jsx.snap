// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Cart component > renders card 1`] = `
<div>
  <div
    class="_card_71d7ae"
  >
    <img
      class="_img_71d7ae"
      src="url"
    />
    <h2
      class="_name_71d7ae"
    >
      title
    </h2>
    <p
      class="_price_71d7ae"
    >
      $
      10
    </p>
  </div>
</div>
`;
